src/docs/product/accounts/sso/okta-sso.mdx

```diff
@@ -10,50 +10,100 @@ description: "Set up Okta Single Sign-On (SSO) on Sentry."
+
+</Alert>
+
+
    - Organization Slug: *YOUR_ORG_SLUG*
-   
+
       - You can find your organization slug in the URL: `https://sentry.io/organizations/YOUR_ORG_SLUG/`
-      
+
    - Email: `Email`
-   
+
    ![Configuring Sentry in Okta](okta-settings-for-sentry.png)
-   
-1. Click "Save", and the Sentry app will be created. 
 
-1. You'll be redirected to the app settings page. Click "Sign On", and you'll be brought back to the same settings in Step 5. This time, click on "View Setup Instructions" in the bright yellow card, and a new tab will open.  
+1. Click "Save" to create the Sentry app.
+
+1. Click "Sign On" to the app settings page to which you are redirected. When the settings displayed in Step 5 appear, select "View Setup Instructions" in the bright yellow card to open a new tab.
 
-1. In the new tab, look for the "Metadata URL" in the setup instructions and copy it.
+1. Copy the "Metadata URL" in the set up instructions.
 
    ![Okta metadata URL in setup instructions](okta-setup-instructions-page.png)
 
-1. Sign in to Sentry. Click **Settings > Auth > Configure Okta**.
-      
+1. Sign in to Sentry. Select **Settings > Auth > Configure Okta**.
+
    ![Sentry Single Sign-On configuration page](okta-sentry-sso-settings-page.png)
-   
-1. Paste the Metadata URL from Step 8, and click "Continue".
+
+1. Paste the Metadata URL from Step 8, then click "Continue".
 
    ![Sentry Single Sign-On configuration for Okta](okta-sentry-settings-metadata-url.png)
-   
-1. Sentry will attempt to authenticate and link your account with Okta. After successful authentication, you'll be redirected to Sentry's SSO configuration page, where you can take the following actions:
 
-   - You can share the "Login URL" value, which will be used for SP-initiated SSO, with the users in your organization.
-   
+Sentry will attempt to authenticate and link your account with Okta. After successful authentication, you'll be redirected to Sentry's SSO configuration page, where you can:
+
+   - Share the "Login URL" value, which will be used for SP-initiated SSO, with the users in your organization.
+
    - Scroll down to the bottom and ensure that "Require SSO" is checked if you want to enforce logging in with Okta.
-   
-   - Set a "Default Role" for new SSO users. Selecting "Member" should cover most use-cases.
-   
-   - If you made changes, click "Save Settings" to complete your setup.
-   
+
+   - Set a "Default Role" for new SSO users. Selecting "Member" should cover most use cases.
+
+If you make any changes, click "Save Settings" to complete your set up.
+
    ![Sentry Single Sign-On settings for Okta](okta-sentry-settings-for-okta.png)
+
+## SCIM Integration
+<Alert level="warning" title="SCIM is not available for general use yet.">
+
+SCIM functionality is currently in early access and is not available to all organizations. Please reach out to support@sentry.io if you'd like to participate in early access.
+</Alert>
```

Comment:
```suggestion
<Note>

This feature is available only if you're in the Early Adopter program. Features available to Early Adopters are still in-progress and may have bugs. We recognize the irony.

If you’re interested in being an Early Adopter, you can turn your organization’s Early Adopter status on/off in General Settings. This will affect all users in your organization and can be turned back off just as easily.

</Note>
```